### PR TITLE
Am profile bug fixes

### PR DESCRIPTION
**Pull request by just-trey**
@SoftFever, There were a few issues once I was able to test the AnkerMake profiles in the final build. This resolves those issues: 

1. When adding as a new  printer, the printer does not appear as an option after adding it in the setup wizard
2. Printer would pause at and of the layer and ooze before proceeding to the next layer. 
3. according to Bambu Lab the internal value for "rectilinear" is actually "zig-zag"
4. reset the resolution to match every other printer

Thanks for all the support!

### CODE REVIEW

File: resources/profiles/Anker/process/0.25mm Fast @Anker.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
     "top_shell_layers": "3",
     "support_bottom_z_distance": "0.25",
     "support_top_z_distance": "0.25",
-    "top_surface_pattern": "rectilinear",
-    "bottom_surface_pattern": "rectilinear",
+    "top_surface_pattern": "zig-zag",
```
Comment (SoftFever) [inline resources/profiles/Anker/process/0.25mm Fast @Anker.json:16]: It's fine as  the applciation will automatically fall back to the default patten.
It's still nice to correct it anyway.

But I would suggest we use `monotonic` for these

File: resources/profiles/Anker/machine/fdm_machine_common.json [hunk 2]
```diff
@@ -110,7 +108,7 @@
     ],
     "silent_mode": "0",
     "single_extruder_multi_material": "1",
-    "use_relative_e_distances": "1",
+    "use_relative_e_distances": "0",
```
Comment (SoftFever) [inline resources/profiles/Anker/machine/fdm_machine_common.json:111]: Relative e is strongly recommended.
I suggest dont change this.

ref: https://github.com/SoftFever/OrcaSlicer/issues/1857

File: resources/profiles/Anker/machine/fdm_machine_common.json [hunk 3]
```diff
@@ -4,12 +4,10 @@
     "from": "system",
     "instantiation": "false",
     "gcode_flavor": "marlin",
-    "machine_start_gcode": "M4899 T3 ; Enable v3 jerk and S-curve acceleration \nM104 S{first_layer_temperature[0]-30} ; Anti-ooze - Set hotend temp to 30 degrees less than target\nM190 S{first_layer_bed_temperature[0]} ; set and wait for bed temp to stabilize\nM109 S{first_layer_temperature[0]} ; set final nozzle temp to stabilize\nG28 ;Home",
+    "machine_start_gcode": "M4899 T3 ; Enable v3 jerk and S-curve acceleration \nM104 Sfirst_layer_temperature[0] ; Set hotend temp\nM190 S{first_layer_bed_temperature[0]} ; set and wait for bed temp to stabilize\nM109 S{first_layer_temperature[0]} ; set final nozzle temp to stabilize\nG28 ;Home",
```
Comment (SoftFever) [inline resources/profiles/Anker/machine/fdm_machine_common.json:7]: 👍

**Review by SoftFever — CHANGES_REQUESTED** [review]

File: resources/profiles/Anker/machine/fdm_machine_common.json (hunk 2 above)
Comment (just-trey) [reply resources/profiles/Anker/machine/fdm_machine_common.json:111]: put back as suggested

File: resources/profiles/Anker/process/0.25mm Fast @Anker.json (hunk 1 above)
Comment (just-trey) [reply resources/profiles/Anker/process/0.25mm Fast @Anker.json:16]: Changed as suggested. I was trying to shave off time, but it is so minimal.

**Review by SoftFever — APPROVED** [review]

**Pull request merged**